### PR TITLE
🎨 Code cleanup, as always

### PR DESCRIPTION
**Pull request by MrKrisKrisu**

### CODE REVIEW

File: app/Http/Controllers/Backend/Admin/EventController.php [hunk 1]
```diff
@@ -37,9 +37,9 @@ private static function createSlug(string $name): string {
 
         $i = "";
         while (Event::where('slug', '=', $slug . $i)->first()) {
-            $i = $i == "" ? 1 : $i + 1;
+            $i = $i === "" ? 1 : $i + 1;
         }
-        if ($i != "") {
+        if ($i !== "") {
```
Comment (HerrLevin) [inline app/Http/Controllers/Backend/Admin/EventController.php]: I'm sorry but wtf?! 
Just use `!empty($i)`? :D

File: app/Http/Controllers/Backend/Export/Format/CsvExportController.php [hunk 2]
```diff
@@ -15,7 +15,7 @@ public static function generateExport(User $user, Carbon $timestampFrom, Carbon
         $data = ExportController::getExportableStatuses($user, $timestampFrom, $timestampTo);
 
         return static function() use ($data) {
-            $fileStream = fopen('php://output', 'w');
+            $fileStream = fopen('php://output', 'wb');
```
Comment (HerrLevin) [inline app/Http/Controllers/Backend/Export/Format/CsvExportController.php]: why force binary when we're writing a text file?

File: app/Http/Controllers/Backend/GeoController.php [hunk 3]
```diff
@@ -111,11 +113,10 @@ public static function calculateDistanceBetweenCoordinates(
 
         $equatorialRadiusInMeters = 6378137;
 
-        $pi       = pi();
-        $latA     = $latitudeA / 180 * $pi;
-        $lonA     = $longitudeA / 180 * $pi;
-        $latB     = $latitudeB / 180 * $pi;
-        $lonB     = $longitudeB / 180 * $pi;
+        $latA     = $latitudeA / 180 * M_PI;
```
Comment (HerrLevin) [inline app/Http/Controllers/Backend/GeoController.php:116]: nice!

File: routes/console.php [hunk 4]
```diff
@@ -12,7 +12,3 @@
 | simple approach to interacting with each command's IO methods.
 |
 */
-
-Artisan::command('inspire', function() {
```
Comment (HerrLevin) [inline routes/console.php:16]: But why? 😢

**Review by HerrLevin — COMMENTED** [review]
I'm not getting paid enough for 62 files.

**Comment by MrKrisKrisu** [comment]
image
Good thing there are only 61. We are still under your limit!

**Review by HerrLevin — APPROVED** [review]

**Pull request merged**